src/extension.ts

```diff
@@ -19,8 +19,10 @@ import * as clipboard from 'clipboardy';
 
 // Internal dependencies
 import { host } from './host';
+import { loadConfigMapData } from './configMap'
 import * as explainer from './explainer';
 import { shell, ShellResult } from './shell';
+import * as configmaps from './configMap';
```

Comment:
Maybe I'm missing something obvious but I can't see `configMap.ts` and when I pull this branch I get missing file errors for it.  Where should I be finding it?

Reply:
added.